server/data/interfaces/curiousApi/LearnerEducation.ts

```diff
@@ -1,75 +1,181 @@
-export default interface LearnerEducation {
```

Comment:
The diff is quite hard to read here, but basically the interface `LearnerEducation` was wrong / misleading.
A "learner education" represents a single course or qualification; whereas what had been defined here was the entire API response.
The API response is a paged collection of learner education records within the `content` field, and then a load of metadata fields representing the page number, size, whether its the last page etc.

As an overall interface it did correctly describe the API response; it's just that the name was wrong. And, given we are using typescript, we can better represent this by breaking out the types.

The change I've made is to verbatim copy and paste the types from the Curious API swagger spec, including their names. Their names are not ideal/descriptive in some cases (eg: their type that represents the whole response is called `Page`), so I have exported them with better, more descriptive and meaningful names.